session/pessimistic_test.go

```diff
@@ -1950,14 +1951,14 @@ func (s *testPessimisticSuite) TestSelectForUpdateConflictRetry(c *C) {
 	tsCh := make(chan uint64)
 	go func() {
 		tk3.MustExec("update tk set c2 = c2 + 1 where c1 = 1")
-		lastTS, err := s.store.GetOracle().GetLowResolutionTimestamp(context.Background())
+		lastTS, err := s.store.GetOracle().GetLowResolutionTimestamp(context.Background(), &oracle.Option{TxnScope: config.DefTxnScope})
```

Comment:
Why not just `&oracle.Option{}`?

Reply:
I think specifying the `TxnScope` here explicitly will be more clear for others to read and understand these codes?

Reply:
Fine. Please define another constant for "global". `config.DefTxnScope` is used to define the default value of a system variable.
e.g.
```
const GlobalScope = "global"     // Defined in another place
const DefTxnScope = GlobalScope
```

Reply:
done.